src/crimson/common/interruptible_future.h

```diff
@@ -111,7 +146,6 @@ auto call_with_interruption_impl(
   // In this case, as crimson::do_for_each would directly do futurize_invoke
   // for "call_with_interruption", we have make sure this invocation would
   // errorly release ::crimson::interruptible::interrupt_cond<InterruptCond>
```

Comment:
Is it the reason that `interrupt_condition` can be null?

Reply:
Hmm, I don't think this comment has explained why `interrupt_condition` is nullable.